packages/client/src/createClient.ts

```diff
@@ -54,6 +54,19 @@ class ActionInvoker<Q extends ActionDefinition<any, any, any>>
   applyAction: (...args: any[]) => any;
 }
 
+class QueryInvoker<Q extends QueryDefinition<any, any>>
+  implements QuerySignatureFromDef<Q>
+{
+  constructor(
+    clientCtx: MinimalClient,
+    queryDef: QueryDefinition<any, any>,
+  ) {
+    this.applyQuery = applyQuery.bind(undefined, clientCtx, queryDef);
+  }
+
+  applyQuery: (...args: any[]) => any;
```

Comment:
Using `applyQuery`, open to other suggestions though. Other ideas I thought of:

- executeQuery
- applyFunction
- executeFunction
Reason I even suggest `function` is because that's what our docs refer to queries as